src/platform/utilities/sso/index.js

```diff
@@ -81,7 +81,12 @@ export async function checkAutoSession(
   profile = {},
 ) {
   const { ttl, transactionid, ...queryParams } = await ssoKeepAliveSession();
-
+  const removeReturnUrl = url => {
+    const isExternalCSP = sessionStorage
```

Comment:
Change variable name to what it does.. something like `shouldRemoveAuthReturnUrl` or `shouldRemoveSessionStorage`